### ISSUE TITLE
Display notifications about new comments to wall posts

### ISSUE THREAD

**Issue by mitar** [wlanslovenija/PiplMesh#165]
For each user, somewhere on the page (next to the user picture) display a button with a number of notifications for this user. Once user clicks on it, a list of notifications is displayed. For now, notifications are about all comments made to wall posts user has interacted with (created or commented it). If user clicks on the notification, that post and comment is put into focus.

**Comment by zupan** [comment]
Work in progress :)

**Comment by mitar** [comment]
Where? Am I missing something?

**Comment by zupan** [comment]
No. I just wanted to say, that i started working on this issue.
